tests/general/InvalidLinkBearTest.py

```diff
@@ -364,24 +382,25 @@ def response(status_code, *args, **kwargs):
         with unittest.mock.patch(
                 'tests.general.InvalidLinkBearTest.requests.head',
                 return_value=response(status_code=200)) as mock:
-            uut = InvalidLinkBear(self.section, Queue())
-            self.assertEqual([x.message
-                              for x in list(uut.run('file', file_contents,
-                                                    network_timeout=nt))], [])
+            self.assertEqual(
```

Comment:
this is extremely overcomplicated garbage.  It can be greatly simplified. using one of the existing helper methods.